docs/build/advanced-concepts/metadata-views.md

```diff
@@ -0,0 +1,226 @@
+
+### Core Properties
+
+Every NFT has core properties that define its basic attributes. For example, in the provided code, an NFT has properties like its unique ID, name, description, and more. When we add the `MetadataViews.Resolver` to our NFT resource, we are indicating that these variables will align with the specifications defined in the `MetadataViews` contract for each of these properties. This ensures interoperability within the Flow ecosystem and guarantees that the metadata of our NFT can be consistently accessed and understood by various platforms and services that interact with NFTs.
+
+```cadence
+pub resource NFT: NonFungibleToken.INFT, MetadataViews.Resolver {
+    pub let id: UInt64
+    pub let name: String
+    pub let description: String
+    pub let thumbnail: String
+    access(self) let royalties: [MetadataViews.Royalty]
+    access(self) let metadata: {String: AnyStruct}
+    ...
+}
+```
+
+### Metadata Views for NFTs
+
+`MetadataViews` types define how the NFT presents its data. When called upon, the system knows precisely which view to render, ensuring that the relevant information is presented consistently across various platforms:
+
+### Display
+
+This view provides basic information about the NFT suitable for listing or display purposes. When the `Display` type is invoked, it dynamically assembles the visual and descriptive information that is typically needed for showcasing the NFT in marketplaces or collections.
+
+```cadence
+case Type<MetadataViews.Display>():
+    return MetadataViews.Display(
+        name: self.name,
+        description: self.description,
+        thumbnail: MetadataViews.HTTPFile(
+            url: self.thumbnail
+        )
+    )
+```
+
+### Editions
+
+The `Editions` view provides intricate details regarding the particular release of an NFT. This can include information about the number of copies in an edition, the sequence number of the specific NFT within that edition, or if it is part of a limited series. When the `Editions` view is queried, it retrieves this data, essential for collectors to understand the rarity and exclusivity of the NFT they are interested in.
+
+```cadence
+case Type<MetadataViews.Editions>():
+    let editionInfo = MetadataViews.Edition(
+        name: "Example NFT Edition",
+        number: self.id,
+        max: nil
+    )
+    return MetadataViews.Editions([editionInfo])
+```
+
+### Serial Number Metadata
+
+The `Serial` metadata provides the unique serial number of the NFT, akin to a serial number on a currency note or a VIN on a car. This serial number is a fundamental attribute that certifies the individuality of each NFT and is critical for identification and verification processes.
```

Comment:
These should be consistent i think. "view type" maybe?
_The `Editions` view type provides_
_The `Serial` view type provides_